### DISCUSSION TITLE
Do supabase provide auto updated_at everytime whenever update query is called on table definition level or through ORM like sequelize ?

### DISCUSSION THREAD

**Discussion by hiteshchandrvadiya** [supabase/supabase#20054] [Questions] [answered]
Hello, 

I want to know whether supabase provide anything that update updated_at everytime when we call the update query, I know the trigger method but I want to know that can we handle it through table definition level or like the sequelize which would update updated_at whenever the update query would be fired. 

So does supabase provide anthing similar to sequelize ORM ?

**Comment by GaryAustin1** [comment, marked as answer]
The trigger method is the way.  There is a helper extension.  https://www.postgresql.org/docs/current/contrib-spi.html#CONTRIB-SPI-MODDATETIME
